sdk/python/kubeflow/training/utils/utils.py

```diff
@@ -193,6 +193,8 @@ def get_container_spec(
     args: Optional[List[str]] = None,
     resources: Union[dict, models.V1ResourceRequirements, None] = None,
     volume_mounts: Optional[List[models.V1VolumeMount]] = None,
+    env: Optional[List[models.V1EnvVar]] = None,
+    env_from: Optional[List[models.V1EnvFromSource]] = None,
```

Comment:
Since Trainer image currently doesn't support env variables to be set by the user can we implement this in the future if that is required ?

Reply:
Yes, of course. I typically don't use these two parameters, but I'm not sure if they are important from the user's perspective. I think we can consider implementing them in the future.